packages/twenty-front/src/modules/settings/data-model/object-details/components/SettingsObjectFieldItemTableRow.tsx

```diff
@@ -38,8 +38,11 @@ type SettingsObjectFieldItemTableRowProps = {
   mode: 'view' | 'new-field';
 };
 
-export const StyledObjectFieldTableRow = styled(TableRow)`
-  grid-auto-columns: 180px 148px 148px 36px;
+export const StyledObjectFieldTableRow = styled(TableRow)<{
```

Comment:
This isn't very clean imo (mix of default + passed value, component seems internal to a file but exported). 
Let's not introduce `$gridColumns`, do two components if we need to, and potentially extract it into a file if it's used in multiple places